docs/markdown/06-iterator_generator/01-Iterator.md

```diff
@@ -54,9 +51,69 @@ for (elem of myArray) {
 
 ##--##
 
-![h-400 center](./assets/images/Iterable_01_array.png) <!-- .element: class="fragment" data-fragment-index="1" -->
+<!-- .slide -->
+
+<br />
+<br />
+
+![h-400 center](./assets/images/Iterable_01_array.png) <!-- .element: class="fragment" -->
 
 Notes:
-à utiliser sur des itérables (not only array ;) )
 
 un iterator renvoie 2 choses, si l'itérateur est fini et la valeur suivante
+{value: 'foo', done: false}
+
+##==##
+
+<!-- .slide: class="with-code" -->
+
+# Itérable : Comment les parcourir
+
+<br />
+
+## Le spread operator
+
+```javascript
```

Comment:
mériterais un petit commentaire pour comprendre pourquoi tu le met au même niveau que le for Of